resources/web/wwi/viewer.js

```diff
@@ -545,7 +549,8 @@ function createIndex(view) {
 
   // Do not create too small indexes.
   const content = document.querySelector('#content');
-  if ((content.offsetHeight < 2 * window.innerHeight || headings.length < 4) && (localSetup.book !== 'discord' || headings.length < 2))
+  if ((content.offsetHeight < 2 * window.innerHeight || headings.length < 4) && (localSetup.book !== 'discord' ||
+    headings.length < 2))
```

Comment:
Better to split at hight level
```suggestion
  if ((content.offsetHeight < 2 * window.innerHeight || headings.length < 4) &&
    (localSetup.book !== 'discord' || headings.length < 2))
```